index.html

```diff
@@ -32,31 +32,168 @@ <h1 class="profile">I'm Damilare Glad to see you!</h1>
                 <ul>
                     <li>
                         <a href="#">
-                         <img src="./images/Twitter-logo.png" class="pix" alt="twitter">
+                         <img src="./images/Twitter-logo.png"  alt="twitter">
                         </a>
                     </li>
                     <li>
                         <a href="#">
-                          <img src="./images/Linkedin-logo.png" class="pix" alt="linkedin">
+                          <img src="./images/Linkedin-logo.png" alt="linkedin">
                         </a>
                     </li>
                     <li>
                         <a href="#">
-                         <img src="./images/M-logo.png" class="pix" alt="letter m">
+                         <img src="./images/M-logo.png" alt="letter m">
                         </a>
                     </li>
                     <li>
                         <a href="#">
-                         <img src="./images/Github-logo.png" class="pix" alt="github">
+                         <img src="./images/Github-logo.png" alt="github">
                         </a>
                     </li>
                     <li>
                         <a href="#">
-                         <img src="./images/fingers-logo.png" class="pix" alt="finger icon">
+                         <img src="./images/fingers-logo.png" alt="finger icon">
                         </a>
                     </li>
                 </ul>
             </section>
         </main>
+        <section class="work-section">
+           <ul class="project">
+                <li class="project-card">
+                  <img src="./images/Snapshoot-Portfolio.png" class="pix" alt="snap">
+                  <h1 class="project-title">TONIC</h1>
+                  <p class="project-description">CANOPY
+                      <span><img src="./images/rename.png" alt="period">Back End Dev </span>
+                      <span><img src="./images/rename.png" alt="period">2015</span>
+                  </p>
+                   <p class="project-content">
+                      A daily selection of privately personalized reads; no accounts or sign-ups required.
+                   </p>
+                   <div class="lang">
+                       <div class="btn">html</div>
+                       <div class="btn">css</div>
+                       <div class="btn">javascript</div>
+                   </div>
```

Comment:
- [ ] I suggest use `ul` tag for the `lang` class and make all the `btn` class child elements `li` or list elements. It is literally a list of languages used in the project. Apply the same changes to the other project sections too.